ui/src/dashboards/reducers/cellEditorOverlay.js

```diff
@@ -19,11 +23,26 @@ export default function cellEditorOverlay(state = initialState, action) {
     case 'SHOW_CELL_EDITOR_OVERLAY': {
       const {cell, cell: {colors}} = action.payload
 
-      const singleStatType = getSingleStatType(colors)
-      const singleStatColors = validateSingleStatColors(colors, singleStatType)
+      const thresholdsListType = getThresholdsListType(colors)
+      const thresholdsListColors = validateThresholdsListColors(
+        colors,
+        thresholdsListType
+      )
       const gaugeColors = validateGaugeColors(colors)
 
-      return {...state, cell, singleStatType, singleStatColors, gaugeColors}
+      const tableOptions = _.get(
+        cell,
+        'tableOptions',
+        initializeOptions('table')
+      )
+
+      return {
+        ...state,
+        cell: {...cell, tableOptions},
```

Comment:
This should probably be done on the serverside, instead of having `tableOptions` be `omitempty`